evaluation/FEEDBACK.md

```diff
@@ -0,0 +1,61 @@
+# Feedback
+
+This document is aiming to aggregate a set of generic rules to make writing good feedback a little easier. Note that it is by far not covering everything.
+
+P.S. Don't forget that common sense goes above all guidelines. Be smart.
+
+## Purpose
+
+Each point in feedback should be aimed to helping receiving person improve. Think about it when writing each of the points.
+
+## Actionable Points
+
+Feedback loses a lot of value if it's not clear what action receiver should take. In most cases it manifests as one of two properties:
```

Comment:
"what action receiver should take" -> "what action the receiver should take"

This sentence would probably be easier if was positive (e.g. "Good feedback should clearly indicate ways to improve.") This would simplify the following sentence too (e.g. "This can usually be achieved in two ways:")